hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/validation/ExpiryMeta.java

```diff
@@ -37,7 +38,32 @@ public record ExpiryMeta(
      * @param autoRenewPeriod the number of seconds between auto-renewals
      */
     public ExpiryMeta(long expiry, long autoRenewPeriod, long autoRenewNum) {
-        this(expiry, autoRenewPeriod, 0, 0, autoRenewNum);
+        this(
+                expiry,
+                autoRenewPeriod,
+                AccountID.newBuilder()
+                        .shardNum(0)
+                        .realmNum(0)
+                        .accountNum(autoRenewNum)
+                        .build());
+    }
+
+    /**
+     * For convenience in tests, creates an instance with the given expiry, auto-renew period,
+     * and auto-renew account number with zeros for the shard and realm of the auto-renew account.
+     *
+     * @param expiry the consensus second at which the entity expires
+     * @param autoRenewPeriod the number of seconds between auto-renewals
+     */
+    public ExpiryMeta(long expiry, long autoRenewPeriod, long autoRenewShard, long autoRenewRealm, long autoRenewNum) {
+        this(
+                expiry,
+                autoRenewPeriod,
+                AccountID.newBuilder()
+                        .shardNum(autoRenewShard)
+                        .realmNum(autoRenewRealm)
+                        .accountNum(autoRenewNum)
+                        .build());
     }
```

Comment:
I think we can delete these two methods

Reply:
There were 78 places using the 1st method and 9 places using the 2nd one. That's why I kept them. Do you really want to delete them?

Reply:
Deleted these two method.